NOTICE

```diff
@@ -7,82 +7,29 @@ listed below. Your use of this material within the component is also subject to
 conditions of these licenses. The LICENSE file contains the text of all the licenses which apply
 within this component.
 
-Copyright (c) 2008-2013 Kristian Hogsberg
-Copyright © 2013 Rafael Antognolli
-Copyright © 2013 Jasper St. Pierre
-Copyright © 2010-2013 Intel Corporation
-Licensed under the MIT license
+Copyright (C) 2014, 2015 Igalia S.L.
+Licensed under the BSD-2 License
+
+Copyright (C) 2012 Intel Corporation. All rights reserved.
+Licensed under the BSD-2 License
+
+Copyright 2016-2019 TATA ELXSI
+Licensed under the Apache License, Version 2.0
+
+Copyright (C) 2015 Metrological
+Copyright (C) 2019 Linaro
+Licensed under the BSD-2 License
 
-xdg-shell-client-protocol.c and xdg-shell-client-protocol.h
 Copyright © 2008-2013 Kristian Høgsberg
 Copyright © 2013 Rafael Antognolli
 Copyright © 2013 Jasper St. Pierre
 Copyright © 2010-2013 Intel Corporation
 Copyright © 2015-2017 Samsung Electronics Co., Ltd
 Copyright © 2015-2017 Red Hat Inc.
-Licensed under the MIT license
-
-Copyright (C) 2006-2015, ARM Limited
-Licensed under the Apache License, Version 2.0
+Licensed under the MIT License
 
-Copyright 2014 Fraunhofer FOKUS
+Copyright (C) 2007 The Android Open Source Project
 Licensed under the Apache License, Version 2.0
 
```

Comment:
My mistake: for consistency, please add:

Copyright (C) 2015,2019 Metrological
Licensed under the BSD-2 License